stroyprombeton/views/catalog.py

```diff
@@ -135,11 +135,17 @@ def get_context_data(self, **kwargs):
             {'name': name, 'url': url()}
             for (name, url) in product_ancestors[offset:offset + limit]
         ]
+        tag_groups = (
+            models.Tag.objects
+            .filter_by_products([product])
```

Comment:
Let's create a todo to implement `filter_by_options` and drop filtering of option's tags  in `product.html`